pulp_smash/api.py

```diff
@@ -89,6 +89,16 @@ def echo_handler(server_config, response):  # pylint:disable=unused-argument
     return response
 
 
+def code_handler(server_config, response):  # pylint:disable=unused-argument
+    """Check the response status code, and return the response.
+
+    :raises: ``requests.exceptions.HTTPError`` if the response status code is
+        in the 4XX or 5XX range.
+    """
+    response.raise_for_status()
+    return response
+
+
```

Comment:
One thing that I would find helpful in the doc string for this handler is a short explanation of why I might choose it versus the `safe_handler`